------------------------------------------------------------------------
r1356348 | rkcosta | 2013-05-28 11:36:38 +0300 (Tue, 28 May 2013) | 11 lines

Merge r1356347 from trunk.

------------------------------------------------------------------------
r1356347 | rkcosta | 2013-05-28 11:32:26 +0300 (Tue, 28 May 2013) | 4 lines

Default arguments should be in the method's declaration.

Not only is this correct C++, but it also fixes the build with clang 3.3.

------------------------------------------------------------------------


Index: kget/transfer-plugins/bittorrent/btsettingswidget.h
===================================================================
--- kget/transfer-plugins/bittorrent/btsettingswidget.h	(revision 1356347)
+++ kget/transfer-plugins/bittorrent/btsettingswidget.h	(revision 1356348)
@@ -19,7 +19,7 @@
 {
     Q_OBJECT
     public:
-        BTSettingsWidget(QWidget * parent, const QVariantList &args);
+        BTSettingsWidget(QWidget * parent = 0, const QVariantList &args = QVariantList());
 
     public slots:
         virtual void load();
Index: kget/transfer-plugins/bittorrent/btsettingswidget.cpp
===================================================================
--- kget/transfer-plugins/bittorrent/btsettingswidget.cpp	(revision 1356347)
+++ kget/transfer-plugins/bittorrent/btsettingswidget.cpp	(revision 1356348)
@@ -17,7 +17,7 @@
 
 KGET_EXPORT_PLUGIN_CONFIG(BTSettingsWidget)
 
-BTSettingsWidget::BTSettingsWidget(QWidget * parent = 0, const QVariantList &args = QVariantList())
+BTSettingsWidget::BTSettingsWidget(QWidget * parent, const QVariantList &args)
   : KCModule(KGetFactory::componentData(), parent, args)
 {
     setupUi(this);

------------------------------------------------------------------------
